src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.State.cs

```diff
@@ -42,7 +42,7 @@ internal abstract class CommonReferenceManager
         internal abstract MetadataReference GetMetadataReference(IAssemblySymbol assemblySymbol);
         internal abstract ImmutableArray<MetadataReference> ExplicitReferences { get; }
         internal abstract ImmutableArray<MetadataReference> ImplicitReferences { get; }
-        internal abstract IEnumerable<KeyValuePair<AssemblyIdentity, PortableExecutableReference>> GetImplicitlyResolvedAssemblyReferences();
+        internal abstract IEnumerable<(PEAssembly, PortableExecutableReference)> GetImplicitlyResolvedAssemblyReferences();
```

Comment:
>PEAssembly [](start = 39, length = 10)

Why do we need to store this explicitly? Can we get this from PortableExecutableReference?